common/gradle.properties

```
@@ -3,6 +3,7 @@ POM_ARTIFACT_ID=common
 POM_ARTIFACT_GROUP_ID=com.mapbox.base
 POM_ARTIFACT_TITLE=Mapbox Common
 POM_DESCRIPTION=Artifact that provides Mapbox module and plugin contracts
-BINTRAY_PROJECT_NAME=common
 
-android.useAndroidX = true
\ No newline at end of file
+REGISTRY_SDK_NAME=com-mapbox-base-common
```

```suggestion
REGISTRY_SDK_NAME=com-mapbox-base-common-android
```

To avoid any confusion. Let's do the same for other artifacts.

---

We already have old artifacts in the SDK registry, if we introduce a new path, we'll have duplicating directories. Agreed with @LukasPaczos to leave as is.